autopts/pybtp/btp/bap.py

```diff
@@ -348,6 +350,33 @@ def bap_broadcast_sink_bis_sync(broadcast_id, requested_bis_sync,
     bap_command_rsp_succ()
 
 
+def bap_add_broadcast_receive_state(broadcaster_addr_type, broadcaster_addr,
```

Comment:
Let's avoid previous mistakes done in BTP BAP and make sure to denote which role this is for; i.e. it should be clear whether this is an action done as the BAP Scan Delegator or the BAP Broadcast Assistant. In this case it's the former, but we don't want to confuse this with `BTP_BAP_CMD_ADD_BROADCAST_SRC` which is for the BAP Broadcast Assistant